### ISSUE TITLE
Poor performance of ABINet on RRC word recognition challenge dataset

### ISSUE THREAD

**Issue by cvarun94** [open-mmlab/mmocr#1128]
Hi, 
I am using the ABINet to train the RRC Focussed Text 2013 challenge dataset. (download link - https://rrc.cvc.uab.es/?ch=2&com=downloads).
But after several tweaks to the config files I was able to train it, but the loss is always hovering around 8 and the predictions are poor.

Any help, tips, suggestions regarding it would be appreciated.
Attaching the config file and the sample prediction for reference.

Thanks

Prediction--
![abinet_50e_test_rrc_recog](https://user-images.githubusercontent.com/15985976/176655915-33997e1e-5e6b-4ed3-8d98-e1627644469c.jpg)

config file--
[varun_abinet_config_30062022.txt](https://github.com/open-mmlab/mmocr/files/9018940/varun_abinet_config_30062022.txt)

**Comment by gaotongxiao** [comment]
Usually, text recognition models are not able to obtain a satisfying performance without sufficient data. A common practice is training the model with ST and MJ datasets and fine-tuning the model on the training set of specific tasks. You might use our pretrained models as your starting point.

**Comment by cvarun94** [comment]
> Usually, text recognition models are not able to obtain a satisfying performance without sufficient data. A common practice is training the model with ST and MJ datasets and fine-tuning the model on the training set of specific tasks. You might use our pretrained models as your starting point.

Hi! Thanks for the tip..
how can I use the pretrained model to finetune it on my custom data? I couldn't find any resource for it..
also, what is the purpose of "repeat" parameter.. I didn't fully understand it.

**Comment by balandongiv** [comment]
Hi @cvarun94 , you can refer this [Unofficial Notebook](https://github.com/balandongiv/mmocr_tutorial/blob/main/custom%20mmocr.ipynb). Note about the `repeat` also included in the `Notebook`

**Comment by cvarun94** [comment]
> Hi @cvarun94 , you can refer this [Unofficial Notebook](https://github.com/balandongiv/mmocr_tutorial/blob/main/custom%20mmocr.ipynb). Note about the `repeat` also included in the `Notebook`

@balandongiv thank you so much.. honestly that unofficial notebook was very helpful.. 

Also,  I have my own custom dataset - cropped images of handwritten dates in different formats.. such as "12-January-2022", " 12-01-2022", "12-Jan-2022" , "12/01/2022" and so on.. so in such cases will I have to provide my custom dictionary of characters? Where can I get resources on training custom data. and related configs to be changed?

Thanks

**Comment by balandongiv** [comment]
I am not that expert, but based from this [discussion](https://github.com/open-mmlab/mmocr/issues/1128#issuecomment-1171857679), I will definitely used the default `'DICT90'

The `unofficial notebook ` should help you understand on how train on your custom dataset.

**Comment by bd20222** [comment]
Hi, @cvarun94 

I just came across similar results as yours, please see my result [here](https://github.com/open-mmlab/mmocr/issues/1166#issuecomment-1230364925). Do you have any finding for the improvement?
